### PR TITLE
Hook helper functions

### PR DESCRIPTION
**Pull request by alisey**

### CODE REVIEW

File: packages/wix-ui-test-utils/src/dom/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './hook';
```
Comment (shlomitc) [inline packages/wix-ui-test-utils/src/dom/index.ts]: you can it the hook.ts directly here, don't think it needs a separate file

**Comment by alisey** [comment]
@shlomitc moved the hook helpers to index.ts

**Comment by shlomitc** [comment]
CI is failing for some reason, I'll run it again

**Comment by shlomitc** [comment]
please merge with master to make CI pass

**Pull request merged**